contracts/token/MintableAndBurnable.sol

```diff
@@ -1,8 +1,8 @@
-pragma solidity ^0.5.10;
+pragma solidity ^0.6.0;
 
 import "./TransferManager.sol";
 
-
+abstract 
 contract MintableAndBurnable is TransferManager {
```

Comment:
Maybe `interface` instead of `abstract`?

Reply:
Abstract is more logic, indeed.

Reply:
`MintableAndBurnable` contract doesn't exist anymore, functions were moved to `TransferManager`